src/main/resources/edu/cornell/kfs/module/purap/cu-spring-purap.xml

```diff
@@ -53,9 +53,6 @@
 	    <property name="digestorRulesFileName">
 	       <value>edu/cornell/kfs/module/purap/util/cxml/b2bPunchOutOrderDigesterRules.xml</value>
 	    </property>
-	    <property name="schemaLocation">
-	       <value>${externalizable.static.content.url}/xsd/purap/b2bPunchOutOrder.xsd</value> 
```

Comment:
Should this schema location be set to the new classpath-based location, as opposed to just removing it entirely?